File: Lab-4.py
# Miguel Madrigal
# Feb. 2023
# Lab 4 

# user greeting
print('This program will let you enter two integer values that show you all the\
npossibble answers.')
print('Please enter ONLY whole numbers!!!')
print()

# user input numbers
fNumber = int(input('Please enter your first number: '))
sNumber = int(input('Please eneter your second number: '))
print() 


# do calculations
addAnswer = fNumber + sNumber
subAnswer = fNumber - sNumber
multAnswer = fNumber * sNumber
divAnswer = fNumber / sNumber
trugAnswer = fNumber // sNumber
modAnswer = fNumber % sNumber
expoAnswer = fNumber ** sNumber


# display the results
print(fNumber, '+' , sNumber, '=', format(addAnswer, ','))
print(fNumber, '-' , sNumber, '=', format(subAnswer, ','))
print(fNumber, '*' , sNumber, '=', format(multAnswer, ','))
print(fNumber, '/' , sNumber, '=', format(divAnswer, ',.2f'))
print(fNumber, '//' , sNumber, '=', format(trugAnswer, ','))
print(fNumber, '%' , sNumber, '=', format(modAnswer, ',')) 
print(fNumber, '**' , sNumber, '=', format(expoAnswer, ','))
print()

# exit message
print('Have a great day :)!!')

File: Lab-4.cpp
#include <iostream>
#include <math.h>
#include <cmath>
#include <iomanip>
using namespace std;

int main()
{
    int fNumber, sNumber;

    cout << "This Program will let you enter two integer values that show you all the possible answers.\n";
    cout << "Please enter ONLY whole numbers!!\n";

    cout << "\nPlease enter your first whole number: ";
    cin >> fNumber;
    cout << "Please enter your second whole number : ";
    cin >> sNumber;

    int addAnswer = fNumber + sNumber;
    int subAnswer = fNumber - sNumber;
    int multiAnswer = fNumber * sNumber;
    float divAnswer = float(fNumber) / float(sNumber);
    int truncAnswer = trunc(divAnswer);
    int modAnswer = fNumber % sNumber;
    double expoAnswer = pow(fNumber, sNumber);
    
    cout << "\n" << fNumber << "+" << sNumber << "=" << addAnswer;
    cout << "\n" << fNumber << "-" << sNumber << "=" << subAnswer;
    cout << "\n" << fNumber << "*" << sNumber << "=" << multiAnswer;
    cout << "\n" << fNumber << "/" << sNumber << "=" << setprecision(2) << fixed << divAnswer;
    cout << "\n" << fNumber << "//" << sNumber << "=" << truncAnswer;
    cout << "\n" << fNumber << "%" << sNumber << "=" << modAnswer;
    cout << "\n" << fNumber << "**" << sNumber << "=" << setprecision(0) << fixed << expoAnswer;

    cout << "\n\nHave a great day :)!!!\n";

    return 0;
